src/styles.css

```diff
@@ -0,0 +1,204 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+body {
+  width: 417px;
+  height: 718px;
+}
```

Comment:
You have done a really great job of fixing the change requested by the previous reviewer. You have evidently put a lot of work into this project 💯 👍. However, on the topic of making your design look like the figma design, there seems to be some extra space and elements on the right side of your design that is not consistent with the figma template which maybe arose in your recent style changes: 
![image](https://user-images.githubusercontent.com/107795045/235281431-a352c196-abd5-4bd6-9144-8a67c299721f.png)

What I think is causing this is the fixed width of 417px on the body in your styles. This is making the body of your design maintain 417px in width regardless of the screen size of the device. You can observe from the screenshot below how the page looks like without the fixed width on the body: 
![image](https://user-images.githubusercontent.com/107795045/235282061-a7fc2a4a-5337-4b9d-aaff-815375021a6e.png)

I would suggest that you remove the fixed width on the body (line 49) so that your design looks consistent with the figma design. 💪 👍